vllm/model_executor/models/phi3v.py

```diff
@@ -272,10 +278,8 @@ def _get_phi3v_image_feature_size(
 
 
 def dummy_data_for_phi3v(ctx: InputContext, seq_len: int):
-    multimodal_config = ctx.get_multimodal_config()
-
-    #TODO: change the logic for dummy data to support dynamic shape
-    _, _, dummy_height, dummy_width = multimodal_config.image_input_shape
+    # TODO: How to get the max possible feature size?
```

Comment:
@Isotr0py does the model have a cap on the maximum feature size? If so, can you figure out what that is? Thanks!

Reply:
In my test, `Phi3v` has a cap maximum `feature size` with 2473 and `fuyu` is 2304 for single image.

For `Phi3v`, the feature size depends on the image size ratio (`hd_num`) whose max cap is 16:1 by default (correspond to 2473 feature size).

For `fuyu`, all input images will be resized to 1920x1080 by default, and then it will re-slice the resized image to unpadded image size or clamp out of box size to 1920/1080. So a 1920x1080 image will cause maximum feature size  with 2473. (And seems that `chameleon` also use a similar image processing logic with `fuyu`)

Generally, I think most of models should have a cap on the maximum feature size, but it depends on the image processing logic and what's the exact target image size.

Reply:
Thanks for the quick response! I'll add them into this PR accordingly.